device/deviceType/deviceType.go

```diff
@@ -0,0 +1,9 @@
+package deviceType
+
+type Type string
```

Comment:
See comment at `device.Type`. It's very confusing to have `device.Type` and `deviceType.Type`.